boards/nxp/mimxrt1180_evk/mimxrt1180_evk_mimxrt1189_cm7.dts

```diff
@@ -34,3 +34,7 @@
 &gpt2 {
 	status = "okay";
 };
+
+&usdhc1 {
+	status = "okay";
```

Comment:
why m7 still defaults enabled?

Reply:
Why can't we set ok status for usdhc1 on both cm33 and cm7 cores?

Reply:
for now it is, but if we enable the dual core, this could misleading to use that we have a usdhc mmu to share usdhc with two core, which is not there.

Reply:
OK,  I have delete ok status for cm7 core.